src/test/lib/storage/storage_manager_test_util.cpp

```diff
@@ -40,34 +40,65 @@ class StorageManagerTestUtil {
     return dictionary_encoded_chunk;
   }
 
-  static std::shared_ptr<Chunk> create_dictionary_segment_chunk_large(const uint32_t row_count,
-                                                                      const uint32_t column_count) {
-    /*
-   * Create a chunk with index-times repeating elements in each segment.
-   * Example: in segment 0 every value is unique, in segment 1 every value appears twice, in segment 2 thrice ...
-   * Dictionary-encode each segment and return dictionary encoded chunk.
-   */
-    auto segments = pmr_vector<std::shared_ptr<AbstractSegment>>{};
-    for (auto segment_index = uint32_t{0}; segment_index < column_count; ++segment_index) {
-      auto new_value_segment = std::make_shared<ValueSegment<int32_t>>(false, ChunkOffset{row_count});
+  static std::vector<std::shared_ptr<Chunk>> get_chunks(
+        const std::string file_name,
+        const uint32_t row_count,
+        const uint32_t column_count,
+        const uint32_t count) {
 
-      auto current_value = static_cast<int32_t>(row_count);
-      auto value_count = uint32_t{1};  //start 1-indexed to avoid issues with modulo operations
-      while (value_count - 1 <
-             row_count) {  //as we start 1-indexed we need to adapt while-condition to create row-count many elements
-        new_value_segment->append(current_value);
-        --current_value;
-        ++value_count;
-      }
+    std::remove(file_name.c_str());
+    const auto chunk = create_dictionary_segment_chunk(row_count, column_count);
+    std::vector<std::shared_ptr<Chunk>> chunks(count);
+    for (auto index = size_t{0}; index < chunks.size(); ++index) {
+      chunks[index] = chunk;
+    }
 
-      auto ds_int =
-          ChunkEncoder::encode_segment(new_value_segment, DataType::Int, SegmentEncodingSpec{EncodingType::Dictionary});
-      segments.emplace_back(ds_int);
+    return chunks;
+  }
+
+  static std::vector<std::shared_ptr<Chunk>> map_chunks_from_file(
+    std::string file_name,
+    uint32_t COLUMN_COUNT,
+    FILE_HEADER read_header) {
+    auto& sm = Hyrise::get().storage_manager;
+    const auto CHUNK_COUNT = sm.get_max_chunk_count_per_file();
+
+    auto mapped_chunks = std::vector<std::shared_ptr<Chunk>>{};
+    for (auto index = size_t{0}; index < CHUNK_COUNT; ++index) {
+      if (index == 0) {
+        mapped_chunks.emplace_back(sm.map_chunk_from_disk(sm.get_file_header_bytes(), file_name, COLUMN_COUNT));
```

Comment:
this could use the constant of the storage manager by declairng this class as friend of the SM
```suggestion
        mapped_chunks.emplace_back(sm.map_chunk_from_disk(StorageManager::_file_header_bytes, file_name, COLUMN_COUNT));
```

Reply:
Best I can offer is to give a method in the test class to call it. We still need a method for this in the SM public interface. Therefore, I'd say we keep it as it is right now.